infrastructure/variables.tf

```diff
@@ -0,0 +1,22 @@
+variable "product" {
+  type    = "string"
+  default = "send-letter-service"
+}
+
+variable "location_api" {
+  type    = "string"
+  default = "UK South"
+}
+
+variable "location_db" {
+  type    = "string"
+  default = "West Europe"
+}
```

Comment:
not used. did you mean to use this one for redis?

Reply:
Removed this.For Redis used the same as api